channeldb/payments.go

```diff
@@ -469,6 +469,19 @@ func (db *DB) FetchSentPayments() ([]*SentPayment, error) {
 	return payments, nil
 }
 
+// DeleteSentPayments deletes all payments from the DB.
```

Comment:
Commit message has this at the bottom w/o context: 
> sq router state machine

Reply:
Done.